client/blocks/login/login-form.jsx

```diff
@@ -498,6 +498,8 @@ export class LoginForm extends Component {
 			return this.renderWooCommerce();
 		}
 
+		const inPartnerCouponFlow = currentQuery?.redirect_after_auth?.includes( 'partnerCoupon' );
```

Comment:
Is it time for us to add the query param name as a constant because we use it in more than 1 place? 😄 

We already have a couple of constants in `client/jetpack-connect/constants.js` (`JETPACK_COUPON_PARTNERS` and `JETPACK_COUPON_PRESET_MAPPING`) - and this it might be something for another PR, but we're starting to touch quite a few different places (login page, connection controller and checkout), so having constants and even helper functions to detect if a query/URL is "partner coupon" starts to make sense